sansio_lsp_client/client.py

```diff
@@ -171,14 +332,47 @@ def parse_request(event_cls: t.Type[Event]) -> Event:
                     " or ServerNotification"
                 )
 
-        if request.method == "window/showMessage":
+        if request.method == "workspace/workspaceFolders":
+            return parse_request(WorkspaceFolders)
+
+        elif request.method == "workspace/configuration":
+            return parse_request(ConfigurationRequest)
+
+        elif request.method == "window/showMessage":
             return parse_request(ShowMessage)
         elif request.method == "window/showMessageRequest":
             return parse_request(ShowMessageRequest)
         elif request.method == "window/logMessage":
             return parse_request(LogMessage)
+
         elif request.method == "textDocument/publishDiagnostics":
             return parse_request(PublishDiagnostics)
+
+        elif request.method == "window/workDoneProgress/create":
+            ev = parse_request(WorkDoneProgressCreate)
```

Comment:
Why is `ev` created here and, as far as i can see, not used anywhere?

Reply:
That line is indeed serves no purpose, my mistake

Reply:
Apparently I already fixed this in #41